tests/e2e_regression/e2e_config.yml

```diff
@@ -1,24 +1,25 @@
 analysis:
   source:
     cache:
-      cache_dir: tests/e2e_regression/data
-      query_on_cache_miss: true
+      cache_dir: tests/e2e_regression/rpc-cache
+      query_on_cache_miss: false
     chain_name: mainnet
     nodes:
       damask:
-        rpc: unix:/tmp/node.sock
+        default:
+          rpc: unix:/tmp/node.sock
     fast_startup: true
   analyzers:
-    metadata_registry:
-      interval: 5m
-    aggregate_stats:
-      tx_volume_interval: 5m
+    # metadata_registry:
+    #   interval: 5m
+    # aggregate_stats:
+    #   tx_volume_interval: 5m
```

Comment:
Note: Our non-block-based analyzers also have a fair bit of complexity in them, and it would be good to test them also. That's a separate task though.

The git-based `metadata_registry` is an exception, but everything else we could fit into this framework. I think what we want to do is have a way to run all the block analyzers to the end, then run all the `evm_tokens_*` and `evm_token_balances_*` analyzers and `aggregate_stats`. We just don't have a way yet of telling them "exit when there is no more work for you".

Reply:
Created https://app.clickup.com/t/866aak668